openml/datasets/functions.py

```diff
@@ -862,7 +869,7 @@ def edit_dataset(
 
       Returns
       -------
-      data_id of the existing edited version or the new version created and published"""
+      Dataset id """
```

Comment:
Sorry if I wasn't clear, I still like the description of which dataset id is returned - just for it not to be referred to as `dataset_id`  which is the internal variable name.

Reply:
The edit only edits the existing dataset now, it does not create a new version.
So that description is outdated.
We could possibly have
"Dataset id of the edited dataset" but its the same as the one we wanted to edit.

Reply:
Thanks! Apologies, I missed that this was the `edit`  function. Is there a reason we even should parse for the id at this point? Wouldn't just parsing the response to verify the edit was done successfully (and if not, raise a clear error) be enough?
If an error has occurred, raise some kind of `OpenMLException` and if everything went OK, do nothing (not even return a dataset id - as it is just the one we provided in the first place?).

Reply:
My bad. I forgot `_perform_api_call` checks for the server messages already.